prow/git/git.go

```diff
@@ -263,6 +263,31 @@ func (r *Repo) Clean() error {
 	return os.RemoveAll(r.dir)
 }
 
+// ResetHard runs `git reset --hard`
+func (r *Repo) ResetHard(commitlike string) error {
+	// `git reset --hard` doesn't cleanup untracked file, adding them to be safe
```

Comment:
You could also `git clean -fd` for more straightforward cleanup (just a suggestion)

Reply:
feels like better to understand than `add -A`, updated the PR to use `clean -df`